tasks/sep/base-003-fp-granite-prestate/.env

```diff
@@ -0,0 +1,6 @@
+ETH_RPC_URL="https://gateway.tenderly.co/public/sepolia"
+SUPERCHAIN_CONFIG_ADDR=0xC2Be75506d5724086DEB7245bd260Cc9753911Be
+OWNER_SAFE=0x0fe884546476dDd290eC46318785046ef68a0BA9
```

Comment:
https://sepolia.etherscan.io/address/0xd6E6dBf4F7EA0ac412fD8b65ED297e64BB7a06E1#readProxyContract#F8

Reply:
DisputeGameProxy: `0xd6E6dBf4F7EA0ac412fD8b65ED297e64BB7a06E1`

Reply:
Confirmed [Owner safe](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L34) and [DisputeGameFactoryProxy](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L50) with superchain-registry as well.